examples/next/pages/ui/components/ai/ai-conversation/attachments.page.tsx

```diff
@@ -19,17 +19,31 @@ function Chat() {
     {
       data: { messages },
       isLoading,
+      hasError,
+      messages: errorMessages,
```

Comment:
@calebpollman maybe we should rethink this API since `messages` exists in the data of the hook. Feels kinda weird to have to rename this messages to errorMessages.